src/Microsoft.Data.Sqlite.Core/breakingchanges.netcore.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "TypeId": "public class Microsoft.Data.Sqlite.SqliteConnection : System.Data.Common.DbConnection",
+    "MemberId": "public virtual System.IntPtr get_Handle()",
+    "Kind": "Removal"
+  },
+  {
+    "TypeId": "public class Microsoft.Data.Sqlite.SqliteDataReader : System.Data.Common.DbDataReader",
+    "MemberId": "public virtual System.IntPtr get_Handle()",
+    "Kind": "Removal"
```

Comment:
@bricelam - The tool removed concept of modified. Modification is combination of Removal of old member & addition of new member. (addition is only added for interface)
See https://github.com/aspnet/BuildTools/issues/154#issuecomment-292321869